### ISSUE TITLE
Apply design to the intro screen - extension detection.

### ISSUE THREAD

**Issue by hbulgarini** [paritytech/parity-bridges-ui#77]
This task is for apply the proper design on the task described #55

**Comment by goldsteinsveta** [comment]
Let's use the same layout and prevent to go to any of the forms unless extension is installed/enabled?
It is also a good place for a friendly illustration:

<img width="1054" alt="Screenshot 2021-04-21 at 14 42 42" src="https://user-images.githubusercontent.com/8782596/115555521-e6cb6d00-a2af-11eb-8b49-a870b8ca7340.png">

**Comment by goldsteinsveta** [comment]
To me it seems like detect accounts part of #55  can live quite comfortably in selectAccounts component, visually and functionally:

<img width="1143" alt="Screenshot 2021-04-21 at 15 55 17" src="https://user-images.githubusercontent.com/8782596/115566081-3d3da900-a2ba-11eb-898f-c049313f7ceb.png">

**Comment by hbulgarini** [comment]
> Let's use the same layout and prevent to go to any of the forms unless extension is installed/enabled?
> It is also a good place for a friendly illustration:
> 
> <img alt="Screenshot 2021-04-21 at 14 42 42" width="1054" src="https://user-images.githubusercontent.com/8782596/115555521-e6cb6d00-a2af-11eb-8b49-a870b8ca7340.png">

love it!

**Comment by hbulgarini** [comment]
> To me it seems like detect accounts part of #55 can live quite comfortably in selectAccounts component, visually and functionally:
> 
> <img alt="Screenshot 2021-04-21 at 15 55 17" width="1143" src="https://user-images.githubusercontent.com/8782596/115566081-3d3da900-a2ba-11eb-898f-c049313f7ceb.png">

In terms of UX i think this might work ... i'm a little bit concerned though about how flexible the material UI component is to display this as a child component. In the past i struggled with this , what do you think?

**Comment by goldsteinsveta** [comment]
@hbulgarini Yeah fair concern... in this case one probably should use base components, rather than more high level ones like menu or similar. But I don't think it's impossible or difficult actually

**Comment by hbulgarini** [comment]
If it's not so time consuming i would try to implement your original design and in case we face incompatibilities we could go to the base component.
